src/main/java/core/basesyntax/FigureBehavior.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface FigureBehavior {
```

Comment:
```suggestion
public interface AreaCalculator {
```

as we can use this interface not only for figures

Reply:
+